bip-0155.mediawiki

```diff
@@ -134,7 +134,7 @@ See the appendices for the address encodings to be used for the various networks
 
 Introduce a new message type <code>sendaddrv2</code>. Sending such a message indicates that a node can understand and prefers to receive <code>addrv2</code> messages instead of <code>addr</code> messages. I.e. "Send me addrv2".
 
-<code>sendaddrv2</code> SHOULD be sent after receiving the <code>verack</code> message from the peer.
+<code>sendaddrv2</code> SHOULD be sent before sending the <code>verack</code> message to the peer.
```

Comment:
I think the SHOULD here should be changed to MUST.

The feature negotiation MUST be completed before the VERSION-VERACK handshake is completed.

Reply:
Changed to `MUST` and reworded to mimic BIP-339 / wtxidrelay.